kafka-ui-react-app/src/lib/yupExtended.ts

```diff
@@ -40,7 +50,17 @@ const isJsonObject = () => {
   );
 };
 
+const isObjectScopeOut = () => {
+  return yup.string().test(
+    'isObjectScopeOut',
+    // eslint-disable-next-line no-template-curly-in-string
+    'That invalid data was filled',
```

Comment:
@Haarolean pls help us with error message

Reply:
```suggestion
    'Schema syntax is not valid',
```

Reply:
@habrahamyanpro is it a generic message for all the errors? If we'd stick with "invalid data" message, that's gonna be ambiguous.